src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts

```diff
@@ -125,7 +125,7 @@ if (platform.isWindows) {
 if (isIPad) {
 	registerSendSequenceKeybinding(String.fromCharCode('C'.charCodeAt(0) - CTRL_LETTER_OFFSET), { // ctrl+c
 		when: ContextKeyExpr.and(KEYBINDING_CONTEXT_TERMINAL_FOCUS),
-		primary: KeyMod.CtrlCmd | KeyCode.KEY_C
+		primary: KeyCode.Ctrl | KeyCode.KEY_C
```

Comment:
Is there any particular reason that we use `KeyCode.Ctrl` here but not `KeyMod.WinCtrl`?

Reply:
you're right, I think it should be winCtrl, fixing